tests/foreman/cli/test_repository.py

```diff
@@ -2423,6 +2426,59 @@ def test_positive_sync_ansible_collection_from_galaxy(self, repo, module_org, mo
         repo = Repository.info({'id': repo['id']})
         assert repo['sync']['status'] == 'Success'
 
+    @pytest.mark.tier2
+    @pytest.mark.upgrade
+    @pytest.mark.parametrize(
+        'repo_options',
+        [
+            {
+                'content-type': 'ansible_collection',
+                'url': ANSIBLE_GALAXY,
+                'ansible-collection-requirements': '{collections: [ \
+                            { name: theforeman.foreman, version: "2.1.0" }, \
+                            { name: theforeman.operations, version: "0.1.0"} ]}',
+            }
+        ],
+        ids=['ansible_galaxy'],
+        indirect=True,
+    )
+    def test_positive_export_ansible_collection(
+        self, repo, module_org, module_product, default_sat
+    ):
+        """Export ansible collection between organizations
+
+        :id: 4858227e-1669-476d-8da3-4e6bfb6b7e2a
+
+        :expectedresults: All content exported and imported successfully
+
+        :CaseLevel: Integration
+
+        :CaseImportance: High
+
+        """
+        import_org = make_org()
+        Repository.synchronize({'id': repo['id']})
+        repo = Repository.info({'id': repo['id']})
+        assert repo['sync']['status'] == 'Success'
```

Comment:
not a blocker - i think that everything up to this point could go to a fixture (it is an env setup)